src/myuw-notifications.html

```diff
@@ -40,6 +40,10 @@
   background-color: var( --myuw-icon-button-hover-bg, rgba(0,0,0,0.2) );
 }
 
+#bell-button:focus {
+  background-color: var( --myuw-icon-button-hover-bg, rgba(0,0,0,0.2) );
+}
+
```

Comment:
Let's change the focus style so that it matches to the behavior of the myuw-help that is already in production. We will have the darker background only applied to the hover state. Can you change what you have here to:

  #bell-button:focus,
  #bell-button:active {
    outline: 2px solid #5e9ed6;
    outline: -webkit-focus-ring-color auto 5px;
  }

Reply:
Done!